src/borg/remote.py

```diff
@@ -647,31 +647,31 @@ def handle_error(error, args):
 
     @api(since=parse_version('1.0.0'))
     def open(self, path, create=False, lock_wait=None, lock=True, exclusive=False, append_only=False):
-        pass
+        """actual remoting is done via self.call in the @api decorator"""
```

Comment:
I think it‘s more helpful than pass. But it could also be a comment and a pass. 

I think adding real documentation about the method here would be bad and it would quickly go stale. We might want to document differences or compatibility constraints that differ from the real Repository class though.